docs/features/nullable-metadata.md

```diff
@@ -0,0 +1,177 @@
+Nullable Metadata
```

Comment:
In this document, consider reflecting actual behavior of the compiler in this PR, excluding any information about possible future changes in this space. This should help to avoid confusion about expectations from this PR.

Reply:
I agree, the state is currently confusing. But I'd rather avoid making extra changes here that will be reverted. Perhaps I could follow up and correct this document if we don't make all the changes in the near future.

---
In reply to: [293915821](https://github.com/dotnet/roslyn/pull/36398#discussion_r293915821) [](ancestors = 293915821)